etc/config/ForwarderCfg.yaml

```diff
@@ -50,3 +50,11 @@ SET_TIMEOUT: 3
 
 # timeout for forwarder to check keepalive of associated CSC
 CHECK_TIMEOUT: 10
+
+# file transfer protocol
+# bbcp is listed as an example. Another option is scp
+# -f, forces the copy
+# -n, does not use DNS to resolve IP address
+# -s, number of streams to use
+XFER_OPTION: bbcp -f -n -s 1 -i ~/.ssh/id_rsa
+#XFER_OPTION: scp -i ~/.ssh/id_rsa
```

Comment:
Consider changing this to use a plug in, vs using a command in the config file.

Reply:
Created follow-up ticket DM-22036 to fix that.